src/daemon/rpc_command_executor.cpp

```diff
@@ -497,48 +496,55 @@ bool rpc_command_executor::show_status() {
     }
   }
 
-  std::time_t uptime = std::time(nullptr) - ires.start_time;
   uint64_t net_height = ires.target_height > ires.height ? ires.target_height : ires.height;
   std::string bootstrap_msg;
-  if (ires.was_bootstrap_ever_used)
+
+  std::ostringstream str;
+  str << "Height: " << ires.height;
+  if (ires.height != net_height)
+      str << "/ " << net_height << " (" << boost::format("%.1f") % get_sync_percentage(ires) << ")";
+
+  if (ires.testnet)     str << " ON TESTNET";
+  else if (ires.devnet) str << " ON DEVNET";
+
+  if (ires.was_bootstrap_ever_used && *ires.was_bootstrap_ever_used && ires.bootstrap_daemon_address)
   {
-    bootstrap_msg = ", bootstrapping from " + ires.bootstrap_daemon_address;
+    str << ", bootstrap " << *ires.bootstrap_daemon_address;
     if (ires.untrusted)
-    {
-      bootstrap_msg += (boost::format(", local height: %llu (%.1f%%)") % ires.height_without_bootstrap % get_sync_percentage(ires.height_without_bootstrap, net_height)).str();
-    }
+      str << boost::format(", local height: %llu (%.1f%%)") % *ires.height_without_bootstrap % get_sync_percentage(*ires.height_without_bootstrap, net_height);
     else
-    {
-      bootstrap_msg += " was used before";
-    }
+      str << " was used";
+  }
+  if (hfres.version < HF_VERSION_PULSE)
+  {
+    if (!has_mining_info) str << ", mining info unavailable";
+    else if (mining_busy) str << ", syncing";
```

Comment:
This check is still worth keeping after the HF, just has a weird misleading name. `mining_busy` appears to be set when Blockchain is synchronizing

Reply:
It feels weird as well to get that from the mining RPC call (and you won't see it at all if you're hitting a public RPC).

`get_sync_percentage()` has this logic in it:
```C++
    float pc = 100.0f * height / target_height;
    if (height < target_height && pc > 99.9f)
      return 99.9f; // to avoid 100% when not fully synced
```
which seems more appropriate to me; I'll wire up the output again based on that, unless you think it wouldn't work as well for some reason.

Reply:
Looks fine to me